public/app/features/admin/UserListAdminPage.tsx

```diff
@@ -0,0 +1,141 @@
+import { fetchUsers, changeQuery, changePage } from './state/actions';
+import { TagBadge } from 'app/core/components/TagFilter/TagBadge';
+
+interface OwnProps {}
+
+interface ConnectedProps {
+  navModel: NavModel;
+  users: UserDTO[];
+  query: string;
+  showPaging: boolean;
+  totalPages: number;
+  page: number;
+}
+
+interface DispatchProps {
+  fetchUsers: typeof fetchUsers;
+  changeQuery: typeof changeQuery;
+  changePage: typeof changePage;
+}
+
+type Props = OwnProps & ConnectedProps & DispatchProps;
+
+const UserListAdminPageUnConnected: React.FC<Props> = props => {
+  useEffect(() => {
+    props.fetchUsers();
+  }, []);
+
+  return (
+    <Page navModel={props.navModel}>
+      <Page.Contents>
+        <>
+          <ActionBar>
+            <Forms.Input
+              size="md"
+              type="text"
+              placeholder="Find user by name/login/email"
+              tabIndex={1}
+              autoFocus={true}
+              value={props.query}
+              spellCheck={false}
+              onChange={event => props.changeQuery(event.currentTarget.value)}
+              prefix={<i className="fa fa-search" />}
+            />
+            <ActionBarSpacing />
+            <Forms.LinkButton href="admin/users/create" variant="primary">
+              New user
+            </Forms.LinkButton>
+          </ActionBar>
+
+          <div className="admin-list-table">
+            <table className="filter-table form-inline filter-table--hover">
+              <thead>
+                <tr>
+                  <th></th>
+                  <th>Login</th>
+                  <th>Email</th>
+                  <th>
+                    Seen&nbsp;
+                    <Tooltip placement="top" content="Time since user was seen using Grafana">
+                      <i className="fa fa-question-circle" />
+                    </Tooltip>
+                  </th>
+                  <th></th>
+                  <th style={{ width: '1%' }}></th>
+                </tr>
+              </thead>
+              <tbody>{props.users.map(renderUser)}</tbody>
+            </table>
+          </div>
+          <Pagination
+            visible={props.showPaging}
+            numberOfPages={props.totalPages}
+            currentPage={props.page}
+            onNavigate={props.changePage}
+          />
+        </>
+      </Page.Contents>
+    </Page>
+  );
+};
+
+const renderUser = (user: UserDTO) => {
+  const editUrl = `admin/users/edit/${user.id}`;
+
+  return (
+    <tr key={user.id}>
+      <td className="width-4 text-center link-td">
+        <a href={editUrl}>
+          <img className="filter-table__avatar" src={user.avatarUrl} />
+        </a>
+      </td>
+      <td className="link-td">
+        <a href={editUrl}>{user.login}</a>
+      </td>
+      <td className="link-td">
+        <a href={editUrl}>{user.email}</a>
+      </td>
+      <td className="link-td">{user.lastSeenAtAge && <a href={editUrl}>{user.lastSeenAtAge}</a>}</td>
```

Comment:
It's potentially bug prone, since in case if `user.lastSeenAtAge === 0`, it will render `<td className="link-td">0</td>`.
To prevent this, consider a not-not syntax, i.e. `{!!user.lastSeenAtAge && <a href ...}`
Pls check out [the proof](https://codesandbox.io/s/condiftional-rendering-not-not-qq6nd)